_posts/2016-09-12-schedule.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Schedule Released!"
+authors:
+ - badboy
+ - hoverbear
+---
+
+At long last, a schedule!
+
+We're proud to announce our lineup for RustFest 2016 in Berlin!
+
+[**Click here to view the schedule!**](/schedule)
+
+# Saturday
+
+Starting at 9:00 and going until 19:00, Saturday is full of talks at the Microsoft Atrium!
```

Comment:
Mentioning that there will be lunch and that it is included in the price. As well as coffee and cold drinks all day long.